### DISCUSSION TITLE
Should the default font size be larger?

### DISCUSSION THREAD

**Discussion by wez** [wezterm/wezterm#517] [Ideas]
Please comment here with the font size that you use in your wezterm configuration, and whether you find the default 10pt font size to be large enough!

**Comment by cxiao** [comment]
I use 11pt Jetbrains Mono, I do find that the default 10pt is a bit too small. FWIW Windows Terminal’s default font size is 12pt.

**Comment by bew** [comment]
I'm at `font_size = 12.0`, 10 seems too small yes, 11 might be a good compromise even though a bit too small for me.

**Comment by wez** [comment]
Alright, I've gone ahead and set the default to 12 points!
